### DISCUSSION TITLE
Can't manage my DNS Records

### DISCUSSION THREAD

**Discussion by bk-info** [vercel/community#6192] [Help] [answered]
### Summary

I have my domain (bought through GoDaddy) hooked up correctly and the nameservers are managed by Vercel. When I go to GoDaddy it says "We can't display your DNS information because your nameservers aren't managed by us."

But here are screenshots of my Project Settings: when I click Edit for each domain, and when I click Advanced on the left-hand side. Nowhere can I manage my DNS records.

<img width="1272" alt="screenshot 1" src="https://github.com/vercel/community/assets/162641390/e442e890-f9d3-4057-829f-610cbca92d87">
<img width="1231" alt="screenshot 2" src="https://github.com/vercel/community/assets/162641390/697f9c7b-4fb6-4ddd-bc1a-a72fcda26c15">
<img width="1289" alt="screenshot 3" src="https://github.com/vercel/community/assets/162641390/1da6a3f0-bad4-45d4-a1db-9c062bf14d5a">

### Steps to Reproduce

- GoDaddy domain
- Vercel nameservers
- Coming back months after the initial setup
- Can't see anywhere I can manage my DNS records

**Comment by amyegan** [comment]
Hey @bk-info. If you go to [vercel.com/dashboard](https://vercel.com/dashboard) and click on the "Domains" tab, you should see your domain listed on that page. From there you can go to the domain's settings page and manage records.

**Reply by bk-info** [reply]
Strange, my domain is definitely connected but it doesn't show up here. 

<img width="1263" alt="Screen Shot 2024-03-20 at 6 40 38 PM" src="https://github.com/vercel/community/assets/162641390/87ebe68e-0cfd-472b-a59d-a4ad398edb1b">

**Reply by bk-info** [reply]
@amyegan my concerns are adding the domain and causing an error because it's already part of the project, or only being able to manage the DNS records by removing the domain and adding it to the project this way which would put our site down for a bit. I can't find any help documentation on navigating this specific issue, do you know the best way to proceed?

Thanks for your help.

**Reply by charlie-ttt** [reply]
@bk-info Did you get this working?  @amyegan Seems like this is still a problem for myself as well.

I have the same problem as well. I configure Godaddy to use Vercel's nameservers `ns1.vercel-dns.com` `ns2.vercel-dns.com`. There is no place where I can configure DNS after the setup. 

Domains doesn't show up on [vercel.com/dashboard](https://vercel.com/dashboard) & "Domains" tab. Only show up in the project setting page, which doesn't have the option to configure DNS. 

![image](https://github.com/vercel/community/assets/38147717/083cd693-8b41-4cab-ae8f-0368a6d18d4a)
![image](https://github.com/vercel/community/assets/38147717/4cc44fe6-3b31-41e7-afe1-b811d6af539b)

**Reply by aamancio** [reply]
I'm having the same issues. I did bought my domain from Vercel and I have literally wasted all day trying to find out where to configure the DNS

**Comment by bk-info** [comment, marked as answer]
Yes, a team member who set this up had to transfer the domain into my team
account. When he'd try transferring it to my personal account it wouldn't
work. I had to go into Settings and find my *Team URL*. Transferring it
that way worked.
